script/release/uploaders/upload.py

```diff
@@ -122,6 +122,9 @@ def main():
     if PLATFORM == 'win32':
       run_python_upload_script('upload-node-headers.py', '-v', args.version)
 
+  if PLATFORM == 'win32':
+    upload_electron(release, os.path.join(OUT_DIR, 'windows_toolchain_profile.json'), args)
```

Comment:
We should use `get_zip_name` to get proper names for this file See https://github.com/electron/electron/blob/cab2361a95dc6700c5fc89c4982d6321efece6d7/script/release/uploaders/upload.py#L35 for usage

Reply:
Additionally, https://github.com/electron/electron/blob/master/script/release/release.js#L97 should be updated to validate the additional files are there for a release.

Reply:
I did not used get_zip_name directly, but it should work as well.